packaging/fedora/snapd.spec

```diff
@@ -593,6 +593,7 @@ popd
 %{_unitdir}/snapd.autoimport.service
 %{_unitdir}/snapd.refresh.service
 %{_unitdir}/snapd.refresh.timer
+%{_unitdir}/snap.mount
```

Comment:
That doesn't look right. I think systemd requires the filename to match the mount path. It won't help to parameterize the contents if we don't do that for the filename too.

Reply:
Yes we spoke about it on IRC. I will fix it now.

Reply:
This should be good now.